nzedb/processing/post/ProcessAdditional.php

```diff
@@ -796,14 +801,13 @@ protected function _getNZBContents()
 		}
 
 		// Get a list of files in the nzb.
-		$this->_nzbContents = $this->_nzb->nzbFileList($nzbContents);
+		$this->_nzbContents = $this->_nzb->nzbFileList($nzbContents, [false, true]);
```

Comment:
This should be [ 0=>false, 1=>true]

Although you wrote you would change from numeric keys to string keys.